.prettierrc

```diff
@@ -0,0 +1,23 @@
+{
+	"printWidth": 110,
+	"useTabs": true,
+	"semi": true,
+	"singleQuote": true,
+	"trailingComma": "none",
+	"bracketSpacing": false,
+	"arrowParens": "avoid",
+	"endOfLine": "lf",
+	"quoteProps": "consistent",
```

Comment:
Please leave all of these as default. I don't want to have to bikeshed any of this, just a clean migration to prettier defaults

Reply:
Thous was adapted to your coding style. Defaults are different.